### PR TITLE
Release PR for 1.1.3

### PR DESCRIPTION
**Pull request by jasmezz**
See changelog

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: lib/WorkflowFuncscan.groovy [hunk 1]
```diff
@@ -45,15 +46,136 @@ class WorkflowFuncscan {
...
+
+        return citation_text
+    }
+
+    public static String toolBibliographyText(params) {
+
+        def preprocessing_text = "<li>Li, H. (2023). bioawk: BWK awk modified for biological data. Github. Retrieved July 12, 2023, from <a href=\"https://github.com/lh3/bioawk\">https://github.com/lh3/bioawk</a></li>"
+
+
+        // TODO DUPLCIATED HMMSEARCH
```
Comment (jfy133) [inline lib/WorkflowFuncscan.groovy:112]:
```suggestion

```

File: workflows/funcscan.nf [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
                             params.amp_hmmsearch_models, params.amp_ampcombi_db,
                             params.arg_amrfinderplus_db, params.arg_deeparg_data,
                             params.bgc_antismash_databases, params.bgc_antismash_installationdirectory,
                             params.bgc_deepbgc_database, params.bgc_hmmsearch_models ]
 for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
 
+
 // Check mandatory parameters
 if (params.input) { ch_input = file(params.input) } else { error("Input samplesheet not specified!") }
+*/
```
Comment (jfy133) [inline workflows/funcscan.nf:30]: I guess we can delete all these commented bits; I'm not sure about the fARGene one and why that's commented out... any ideas/remember why?

File: workflows/funcscan.nf [hunk 3]
```diff
@@ -117,16 +126,18 @@ workflow FUNCSCAN {
...
-    //
+    /*
     // SUBWORKFLOW: Read in samplesheet, validate and stage input files
     //
     INPUT_CHECK (
-        ch_input
+        file(params.input)
     )
     ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
+*/
```
Comment (jfy133) [inline workflows/funcscan.nf:136]: Can remove now

**Review by jfy133 — APPROVED** [review]
one-handed review featuring sleeping baby in my arm!

File: workflows/funcscan.nf (hunk 2 above)
Comment (jasmezz) [reply workflows/funcscan.nf:30]: I never touched them, probably @louperelo once needed them to check for invalid arg classes input?

**Pull request merged**